cloverage/src/cloverage/report.clj

```diff
@@ -216,9 +216,10 @@
         partial    (total :partial-lines)
         lines      (total :instrd-lines)
         cov-forms  (total :covered-forms)
-        forms      (total :forms)]
-    {:percent-lines-covered (* (/ (+ covered partial) lines) 100.0)
-     :percent-forms-covered (* (/ cov-forms forms) 100.0)}))
+        forms      (total :forms)
+        line-total (+ covered partial)]
+    {:percent-lines-covered (if (= line-total 0) 0. (* (/ line-total lines) 100.0))
```

Comment:
Surely this (and the next line) should check if `lines` is 0, not `line-total`?

I guess it's functionally the same (as `line-total` should never be > `lines`) but it obscures the meaning somewhat IMO